pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>3.2.9</version>
+		<version>3.4.3</version>
```

Comment:
nit: you should've probably picked a shorter commit header with bigger body.

I'd rephrase it like that:
```
Fix: address dependencies issues

- Add missing versions of libraries
- Add missing dotenv dependency
- Update Spring Boot version
```

Maybe it would be worth splitting this commit into several ones, but not sure how it works in Java, so I leave it up to you